sources/packages/api/src/route-controllers/education-program-offering/education-program-offering.controller.ts

```diff
@@ -157,7 +161,12 @@ export class EducationProgramOfferingController extends BaseController {
         "Not able to find a Education Program Offering associated with the current Education Program, Location and offering.",
       );
     }
-    return transformToProgramOfferingDto(offering);
+    return transformToProgramOfferingDto({
+      ...offering,
+      locationName: offering.institutionLocation.name,
```

Comment:
Is this information used to populate the common header?
If yes, should it not be handled inside the common header with another API call?

Reply:
Modified the order of the submitted and Institution Name as discussed.